### ISSUE TITLE
[Accessibility] Button label does not clearly convey its purpose

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#324]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-419) to this repo.

- EQD Audit ID: WooExt-514
- Component/Page: Single Products
- Affected Element: Amazon Pay button
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/product/lightweight-wool-table/

## WCAG Compliance

- Success Criterion: 1.3.1 Info and Relationships
- WCAG Version: 2
- Level: A
- Severity: Medium

## Issue Description

Buttons should have labels or accessible names that clearly describe their purpose. The current label is unclear and causes confusion, failing to accurately convey the button's function.
The screen reader only announces "Amazon Pay"  and does not read the helper or description text: ‘Use your Amazon Pay Sandbox test account’. As a result, the button’s purpose may be unclear.

### Impacted Populations

Blind and low vision people, Screen reader users (blind/low vision and neurodivergent people)

## Recommended Fix

As mentioned in WooExt-509, this image of text should be replaced with actual text. After replacing the image with actual text, use aria-describedby to connect the helper text with the button.

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby

### Code Example
```
<div class="amazonpay-button-container amazonpay-button-enabled amazonpay-button-container-rows" id="maxo-button-wrapper-1764832198951"><div class="amazonpay-button-view1 amazonpay-button-view1-gold" tabindex="0"><picture class="amazonpay-button-logo"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/logo._CB452516594_.svg" alt="Amazon Pay"></picture><picture class="amazonpay-button-sandbox-logo"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/sandbox_icon._CB452516595_.svg" alt=""></picture><picture class="amazonpay-button-chevrons"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/AmazonPay_button_chevron._CB1558391205_.svg" alt=""></picture></div><div class="amazonpay-button-view2"></div><div class="amazonpay-button-view3"><picture class="amazonpay-button-microtext"><img src="https://static-na.payments-amazon.com/assets/maxo/microtext/en_US.svg" aria-label="Use your Amazon Pay Sandbox test account"></picture></div></div>
```
## Screenshots

https://drive.google.com/file/d/1cP_lThTz1KxxAsD8VphK_ps7H23--7TO/view?usp=drivesdk

**Comment by ChristianAF** [comment]
Hey @amberhinds, this issue is not valid for the plugin. It's related to the Amazon SDK. We can not handle the Amazon Pay button rendering.

**Comment by amberhinds** [comment]
Hi @ChristianAF, can you report upstream to Amazon or tell me how?

**Comment by ChristianAF** [comment]
the SDK repo we use is this one https://github.com/amzn/amazon-pay-api-sdk-php . You can open all the button rendering accessibility issues there. The ones for the sites, I need to ask.
